versions/3.0.md

```diff
@@ -763,7 +763,7 @@ Describes a single API operation on a path.
 Field Name | Type | Description
 ---|:---:|---
 <a name="operationTags"></a>tags | [`string`] | A list of tags for API documentation control. Tags can be used for logical grouping of operations by resources or any other qualifier.
-<a name="operationSummary"></a>summary | `string` | A short summary of what the operation does. For maximum readability in editing or documentation generation tools, this field SHOULD be less than 120 characters.
```

Comment:
@webron @DavidBiesack during the TDC call I was asked to make this a 2-part PR, one to remove the character limitation and the second to improve the guidance, esp to better clarify the difference btw `summary` and `description`. Please find that here: https://github.com/OAI/OpenAPI-Specification/pull/1095, and thanks for the comments.